bgpd/bgpd.c

```diff
@@ -7509,6 +7509,39 @@ int peer_ttl_security_hops_unset(struct peer *peer)
 	return ret;
 }
 
+static void peer_reset_message_stats(struct peer *peer)
+{
+	time_t now;
+
+	if (peer) {
+		time(&now);
```

Comment:
Btw, we should use monotime(). Just the same as bgp_clock() does (CLOCK_MONOTONIC).

Reply:
I took the reset of the `update_time` value out of the code. As Donald mentioned it does not make much sense to reset this timer value.